finals-2024-01-16/code/5089-jovan-karajanov/src/app/sort-pipe/pipe.ts

```diff
@@ -0,0 +1,27 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'sort'
+})
+export class SortPipe implements PipeTransform {
```

Comment:
Extra +5 points for the pipe - however, the way you are using it, it's not needed, the band data is already sorted in the component code :smile: